### PR TITLE
chore(stock): remove abherrant AMC/CMM algorithms

### LINKED ISSUE #5343: Remove aberrant the CMM algorithms
As pointed out by @jmcameron [pointed out](https://github.com/IMA-WorldHealth/bhima/pull/5317), the CMM algorithms 1 & 2 are much different than the algorithms 3 & 4.  That is because I made them up as examples in my explanation of the impact of different variables on CMM (see the presentation made to the team last summer).  They worked their way into the final build somehow.

Let's remove them entirely.  They aren't backed by literature and may produce really poor results.  They were useful as sanity checks, but should not be used in production.

See https://github.com/IMA-WorldHealth/bhima/issues/4864#issuecomment-682446809 for more information.

### PR DESCRIPTION
**Pull request by jniles**
This commit completely removes algorithms 2 & 3 from the CMM calculations.  They were originally for demonstration purposes and really shouldn't have made it into production code.  Any sites using one of the abherrant algorithms will be reset to using the MSH algorithm.

Closes #5343.

---

Here is what the AMC/CMM modal looks like now:

image

### CODE REVIEW

File: client/src/i18n/fr/stock.json [hunk 1]
```diff
@@ -142,10 +142,6 @@
       "AVERAGE_MONTHLY_CONSUMPTION"  : "Consommation Mensuelle Moyenne",
       "ALGO_1"          : "Algorithme 1",
       "ALGO_1_COMMENT"  : "La consommation moyenne mensuel est obtenue en divisant la quantité consommée durant la période définie par le nombre de jour de stock pour ladite période, et en multipliant le résultat obtenu par 30.5",
-      "ALGO_2"          : "Algorithme 2",
-      "ALGO_2_COMMENT"  : "La consommation moyenne est obtenue en divisant la quantité consommée durant la période définie par le nombre de jours de consommation pour ladite période, et en multipliant le résultat obtenu par 30.5",
-      "ALGO_3"          : "Algorithme 3",
-      "ALGO_3_COMMENT"  : "La consommation moyenne est obtenue en divisant la quantité consommée durant la période définie par le nombre des jours de la période, et en multipliant le résultat obtenu par 30.5",
       "ALGO_4"          : "Algorithme 4 (MSH)",
       "ALGO_4_COMMENT"  : "La consommation moyenne est obtenue en divisant la quantité consommée durant la période définie par le nombre de mois moins le nombre de mois divisé par le total de nombre de jour de rupture",
```
Comment (mbayopanda) [inline client/src/i18n/fr/stock.json]: L'explication de l'algorithme MSH est un peu difficile à lire actuellement, que pensez-vous de : 
```
La consommation moyenne est égale à : 

La quantité consommée durant la période / (Nombre de mois à considérer - (Le nombre de jour de rupture de stock / 30.5)). 

L'algorithme MSH est recommandé par l'organisation Management Sciences for Health (https://www.msh.org).
```

**Review by mbayopanda — APPROVED** [review]
@jniles, the cleansing of Algo2 and Algo3 has been done perfectly 👍🏽 

I add a non-blocking comment about typo of Algo 4, and also when all consumption are zero we have this : the consumed quantity is not displayed
 
image

File: client/src/i18n/fr/stock.json (hunk 1 above)
Comment (jniles) [reply client/src/i18n/fr/stock.json]: Thanks!  This is a great change.  I'll make it.  👍

**Comment by jniles** [comment]
@mbayopanda thanks for this review!  I'll take a look at that case of 0 values.  I don't think I've tested it before.

**Comment by jniles** [comment]
Made the changes, got approval and passing tests.  Gonna release this before our next BHIMA release.

bors r+

**Pull request merged**